.github/workflows/release.yml

```diff
@@ -81,6 +83,15 @@ jobs:
               --key-name default \
               --key-value ${{ env.AZ_FXNS_DEFAULT_KEY }}
 
+      - name: Set Azure Function SimpleReport Key
+        run: |
+              az functionapp function keys set \
+              --resource-group ${{ env.RESOURCE_GROUP }} \
+              --name ${{ env.RESOURCE_GROUP }} \
+              --function-name reports \
+              --key-name SimpleReport \
+              --key-value ${{ env.AZ_FXNS_SIMPLEREPORT_KEY }}
```

Comment:
Hmm.   This seems OK for SimpleReport, but it doesn't scale to have to change our git workflow every time we onboard or offboard a client of the hub.  Still, given that we plan to switch away from keys to a better auth mechanism, this seems ok for now.

Reply:
I agree - this was a stop-gap fix until we have a secure storage spot for authentication credentials.